Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'andand'
 gem 'truncate_html'
 gem 'representative_view'
 gem 'rabl'
-gem "active_model_serializers"
+gem "active_model_serializers", "0.8.4"
```

Comment:
It would be nice to add a comment here why we chose this version and a hint when to upgrade (Rails version).